sources/packages/backend/apps/db-migrations/src/sql/Types/Create-application-offering-change-request-status-types.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TYPE sims.application_offering_change_request_status_types AS ENUM (
+  'In progress with the student',
```

Comment:
As per the second AC in the ticket `the` is not present here. But first AC has.

@hellolynn-tbtb ? which one should we go for?

Reply:
this is what I had in my story, https://app.zenhub.com/workspaces/student-information-management-system-5fce9df5aa1b45000e937014/issues/gh/bcgov/sims/1953. Open for suggestion @hellolynn-tbtb

Reply:
@ann-aot please use the Figma one and adjust the ticket.
![image](https://github.com/bcgov/SIMS/assets/61259237/090385cd-be30-41c0-aa41-c3a20cab3c3e)

Reply:
as @hellolynn-tbtb  confirmed going without `the`